body-class.js

```diff
@@ -1,5 +1,9 @@
 const FlowRouter = Package['kadira:flow-router'] ? Package['kadira:flow-router'].FlowRouter : false;
 
+function _replaceDot(routeName) {
+  return routeName.replace(/\./gi, "-");
```

Comment:
I've sticked with single quotes around strings in this package, so that'd be great.